spec/features/projects_spec.rb

```diff
@@ -19,20 +19,23 @@
     Organization.create!(name: 'Etsy').projects.create!(name: 'foo_php', tags: nil)
 
     visit '/'
-
-    expect(page).to have_content('Out of sync')
-    expect(page).to have_content('Foo Php')
-    expect(page).to have_content('Shipping')
-    expect(page).to have_content('Scheduling')
+    expect(page).to have_selector 'div[id="projects_data"]'
+    props = find('div[id="projects_data"]')["data-props"].as_json
+    expect(props).to have_content('Foo Php')
+    expect(props).to have_content('Shipping')
+    expect(props).to have_content('Scheduling')
```

Comment:
Is it no longer possible to assert against the rendered page content given these changes?

Reply:
I'm guessing this relates to hydration, I don't think we are server side rendering. All the view logic is tested on the Jest side, so repurposed this test to confirm that `props` are present and that actioncable updates are pushed to props.

Reply:
Yeah, _if_ we wanted to get this working we'd likely need to set up JS testing in Capybara using whatever helpers are current. (A year ago it was the [Webdrivers](https://github.com/titusfortner/webdrivers) gem, not sure currently).

Reply:
Yeah, would love to work on improving this js browser testing at some point! Once the first one is setup, it's really not too bad to add additional full outer-loop tests and they can be quite helpful.

(And yes, that webdrivers gem is still the best practice these days @anandaroop )